src/lib.rs

```diff
@@ -12,7 +12,6 @@
 #![allow(clippy::type_complexity)]
 #![allow(clippy::unit_arg)]
 #![deny(clippy::clone_on_ref_ptr)]
-#![deny(clippy::trivially_copy_pass_by_ref)]
```

Comment:
I suppose we remove it here since it reports a false positive? Which code conflicts with that rule?